src/main/java/io/aiven/kafka/auth/AivenAclAuthorizer.java

```diff
@@ -60,6 +63,8 @@ public boolean reloadAcls() {
     public void configure(final java.util.Map<String, ?> configs) {
         configFileLocation = (String) configs.get("aiven.acl.authorizer.configuration");
         jsonReader = new AclJsonReader(Paths.get(configFileLocation));
+        auditor = AuditorFactory.create(configs);
```

Comment:
I think we can better utilize Kafka's configuration tooling here by:
1. Defining configs for `AivenAclAuthorizer` and auditors.
2. Using `AbstractConfig.getConfiguredInstance` ([some reference](https://github.com/apache/kafka/blob/4ac892ca783acab8e574b9b24d17e767eedb3d5f/connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java#L364)).

We'll get a more formal definition of the configuration, some standartized config checks for free, instantiation, etc.